### PR TITLE
Fix: incorrectly filtered events with compact=true

### PR DESCRIPTION
**Pull request by ixrock**
Since #2113 we're passing items to `KubeObjectList` from above and the items must be sorted in advance to work with `compact=true`

_Before:_
image

_After:_
image

_Events page:_
image

### CODE REVIEW

**Review by Nokel81 — COMMENTED** [review]
This is moving quite fast for me, what is "compact" mode supposed to do? If it is merely displaying the entries in a compact mode. What does that have to do with sorting. I think this is the wrong direction to take.

**Comment by ixrock** [comment]
> This is moving quite fast for me, what is "compact" mode supposed to do? If it is merely displaying the entries in a compact mode. What does that have to do with sorting. I think this is the wrong direction to take.

"Compact-mode" is used only in Workloads overview..  Please see for more details #2113 #2123
This is done because items sorting is handled inside `ItemsListLayout` currently (yes, this whole component bloated and needs refactoring, but this is another story).

File: src/renderer/components/+events/events.tsx [hunk 1]
```diff
@@ -37,12 +40,20 @@ const defaultProps: Partial<Props> = {
 export class Events extends React.Component<Props> {
   static defaultProps = defaultProps as object;
 
-  get store() {
+  private defaultSorting: TableSortParams = {
+    sortBy: columnId.type,
+    orderBy: "desc", // show "Warning" events at the top
```
Comment (jim-docker) [inline src/renderer/components/+events/events.tsx:45]: Sorting by type here corresponds to what's done on the Events page but subsequent sorting via the UI doesn't work the same. For age sorting on the workloads overview page it only sorts by age the 10 events that were first sorted by type.

**Review by jim-docker — APPROVED** [review]

File: src/renderer/components/+events/events.tsx [hunk 2]
```diff
@@ -37,12 +40,20 @@ const defaultProps: Partial<Props> = {
 export class Events extends React.Component<Props> {
   static defaultProps = defaultProps as object;
 
-  get store() {
+  private defaultSorting: TableSortParams = {
+    sortBy: columnId.type,
```
Comment (jakolehm) [inline src/renderer/components/+events/events.tsx:44]: Missed this in #2113 but shouldn't we sort events by "age"?

**Review by aleksfront — APPROVED** [review]

**Comment by ixrock** [comment]
closing in favour #2141

**Pull request closed without merging**